src/main/java/org/sciborgs1155/robot/drive/Drive.java

```diff
@@ -161,6 +161,10 @@ public Command brake() {
     return run(() -> setChassisSpeeds(new ChassisSpeeds()));
   }
 
+  public double angularVelocity() {
+    return gyro.getRate();
```

Comment:
this won't work in sim because our gyro isn't simulated. use a ChassisSpeeds object from drive instead